build-tools/cmake/xa_macros.cmake

```diff
@@ -77,3 +77,52 @@ macro(xa_common_prepare)
       )
   endif()
 endmacro()
+
+macro(xa_macos_prepare_arm64)
+   if(APPLE)
+     set(SDK_SUPPORTS_ARM64 False)
+     set(SDK_SUPPORTS_X86_64 False)
+     execute_process(
+       COMMAND xcode-select -p
+       RESULT_VARIABLE XCODE_SELECT_RESULT
+       OUTPUT_VARIABLE XCODE_DEVELOPER_PATH
+     )
+     if(NOT ${XCODE_SELECT_RESULT} EQUAL "0")
+       message(WARNING "xcode-select failed with result ${XCODE_SELECT_RESULT}")
```

Comment:
Shouldn't this *also* include `{XCODE_DEVELOPER_PATH}`, which contains stdout & stderr?  Or will stdout & stderr be printed *anyway*?

Reply:
stdout won't be printed, stderr will. If `xcode-select` failed then `XCODE_DEVELOPER_PATH` will be empty anyway:

```
$ xcode-select -pp 2> /dev/null
$ xcode-select -pp > /dev/nullxcode-select: error: invalid argument '-pp'
Usage: xcode-select [options]
```